columnflow/production/cms/pdf.py

```diff
@@ -74,4 +74,26 @@ def pdf_weights(self: Producer, events: ak.Array, **kwargs) -> ak.Array:
     events = set_ak_column_f32(events, "pdf_weight_up", 1 + stddev)
     events = set_ak_column_f32(events, "pdf_weight_down", 1 - stddev)
 
+    if ak.any(stddev > 0.5):
```

Comment:
Could you store `stddev > 0.5` as e.g. `outlier_mask = stddev > 0.5` and use it in L80?